src/compose/ComposeBox.js

```diff
@@ -510,7 +513,7 @@ class ComposeBox extends PureComponent<Props, State> {
             />
           </View>
           <FloatingActionButton
-            accessibilityLabel="Send message"
+            accessibilityLabel={_('Send Message, {accessibilityLabel}', { accessibilityLabel })}
```

Comment:
This should say something like `Send message to: {narrowDescription}`. I'm hoping that

1. This makes plenty of sense to English speakers, and
2. Translators can see that string and recognize what it means (remember, they don't generally see the code!), and
3. Translators can find a way to render it in the target language.